.github/workflows/rust.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       matrix:
         rust:
+          - stable
           - nightly
-          - nightly-2019-05-21  # MSRV
```

Comment:
May I suggest we have another MSRV test target for the first Rust stable version that can compile Rocket?

Reply:
Not a bad idea. We should float the nightly version and let it break as-is and make sure we put Rust 1.45.0(?) as the MSRV for Rocket 0.5 on stable.

Reply:
If there is a MSRV requirement for Rocket, this totally makes sense. Any idea where I could find this?
In its GitHub workflow, Rocket is only testing against stable and nightly: https://github.com/SergioBenitez/Rocket/blob/45b4436ed3a7ab913d96c2b69ee4df7fd8c0c618/.github/workflows/ci.yml

Reply:
Let's set it to the current stable 1.46. I am not sure when the features that Rockets use were stabilised. If you know it, we can set it to that version.

Reply:
Chosing an arbitrary version for forcing compatibility feels strange to me. What would be the benefit of this?
I will test building Rocket with different versions later to determine the corresponding version - but even then would using this constrain rocket_cors more strictly than Rocket itself.

Reply:
I see what you mean. Rocket used to have a "supported" Rust nightly version. It does not seem to have one right now for Stable. I guess we can omit this for now until Rocket has a MSRV.

Reply:
For Rocket 0.5, the minimum stable Rust version should be 1.45. The one before the current (1.46.0) release. Either way, I’m fine with not setting that now until Rocket 0.5, itself, stabilizes. Afterwards, we can pin a version for compatibility.

Reply:
> I’m fine with not setting that now until Rocket 0.5, itself, stabilizes. Afterwards, we can pin a version for compatibility.

Sounds like a plan! I created #83 for this, so we don't forget.